### PR TITLE
stackrox: ocp 4.8 variant for master builds

### PR DESCRIPTION
**Pull request by BradLugo**
/hold
/uncc
/assign @BradLugo

### CODE REVIEW

**Review by gavin-stackrox — COMMENTED** [review]
WDYT about removing the `run_if_changed` lines and remove the `postsubmit: true` jobs? I _think_ based on conversations we no longer have to cover OCP pre 4.10 for ACS 4.xx (which is current master). cc @msugakov 
Though I think its wide to keep the option of running them on a PR.

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-8.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- as: ui-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      TEST_SUITE: ocp-ui-e2e-tests
+    test:
+    - ref: stackrox-e2e
+    workflow: ipi-aws
+  timeout: 5h0m0s
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-8.yaml:54]: lets drop this
```suggestion
```

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Comment by BradLugo** [comment]
Other PRs covered these changes

**Pull request closed without merging**